.direnv/flake-profile

```diff
@@ -0,0 +1 @@
+flake-profile-1-link
```

Comment:
irrelevant to the exercise, please remove. also .direnv will be removed from the project as it should be .gitignored.